app/models/exchange_rates/period.rb

```diff
@@ -4,4 +4,6 @@ class ExchangeRates::Period
   include ApiEntity
 
   attr_accessor :year, :month
+
+  has_many :files, class_name: 'ExchangeRates::File'
```

Comment:
Do you need the nesting here? I can't remember. I guess this just gets constantized?

Reply:
yeah, I need it here